tests/e2e/pageobjects/ide/OpenWorkspaceWidget.ts

```diff
@@ -66,9 +66,13 @@ export class OpenWorkspaceWidget {
 
     async selectRootWorkspaceItemInDropDawn(rootProject: string) {
         Logger.debug(`OpenWorkspaceWidget.selectRootWorkspaceItemInDropDawn "${rootProject}"`);
-
-        await this.driverHelper.waitAndClick(By.css(OpenWorkspaceWidget.THEIA_LOCATION_LIST_CSS));
-        await this.driverHelper.waitAndClick(By.css(`option[value=\'file:///${rootProject}']`));
+        if (rootProject.length === 0 || rootProject === '/') {
+            await this.driverHelper.waitAndClick(By.css(OpenWorkspaceWidget.THEIA_LOCATION_LIST_CSS));
```

Comment:
This line command is common for both if and else statements - so it can be placed prior to the `if`.

Reply:
Tnx! replaces by if